packages/blocks/tests/php/test-blocks.php

```diff
@@ -0,0 +1,69 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Test methods from Automattic\Jetpack\Blocks
+ *
+ * @since 9.0.0
+ *
+ * @package automattic/jetpack-blocks
+ */
+
+namespace Automattic\Jetpack;
+
+use Automattic\Jetpack\Blocks;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class Test_Blocks
+ */
+class Test_Blocks extends TestCase {
+	/**
+	 * Test the different inputs and matching output for Classes.
+	 *
+	 * @since 9.0.0
+	 *
+	 * @covers Automattic\Jetpack\Blocks::classes
+	 */
+	public function test_block_classes() {
+		$block_name = 'foo';
+		$attr       = array(
+			'bar'       => 'baz',
+			'align'     => 'wide',
+			'className' => 'editorclass',
+		);
+		$extra      = array( 'extraclass' );
+
+		$block_classes = Blocks::classes( $block_name, $attr, $extra );
+
+		$this->assertContains( 'wp-block-jetpack-foo', $block_classes ); // a general class is created from the block name.
+		$this->assertNotContains( 'bar', $block_classes ); // The extra 'foo' attribute should be dropped.
```

Comment:
There's no "foo" attribute. There's a "bar" attribute with value "baz", should both of those be tested for not being included?

Reply:
Good point. Updated in 29329d52291b34254ead32f367a6d80f9942dc30